js-miniapp-sdk/README.md

```diff
@@ -185,6 +183,17 @@ miniApp.user.getProfilePhoto()
 	});
 ```
 
+`getContacts()`:  Returns the [Contact](api/interfaces/contact.md) list from the Host app.
+
+```javascript
+MiniApp.user.getContacts()
```

Comment:
`MiniApp` or `miniApp`? Because we can keep consistent format while naming it like user name / profile photo.